src/main/java/FRIDAY/Storage.java

```diff
@@ -0,0 +1,107 @@
+package FRIDAY;
+
+import java.io.*;
```

Comment:
Wildcard imports are a violation of the coding standard, I suggest import every class your require individually.